pkg/api/api.go

```diff
@@ -467,6 +467,7 @@ func (hs *HTTPServer) registerRoutes() {
 
 		// Dashboard
 		apiRoute.Group("/dashboards", func(dashboardRoute routing.RouteRegister) {
+			dashboardRoute.Delete("/", authorize(reqSignedIn, ac.EvalPermission(dashboards.ActionDashboardsDelete)), routing.Wrap(hs.DeleteDashboardsByTagsORUIDs))
```

Comment:
Please confirm the correct camel casing here: `DeleteDashboardsByTagsORUIDs` 
`ORUIDs --> OrUIDs`

Reply:
Done